docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,210 @@
+```
+
+And the service `http-server` is stopped. We can verify it by viewing all the services:
+
+```bash
+$ pebble services
+Service      Startup  Current   Since
+http-server  enabled  inactive  today at 11:33 UTC
+```
+
+And, if we run `curl localhost:8080`, we get a "connection refused" error, which confirms the service is down.
+
+To start it again, run:
+
+```bash
+$ pebble start http-server
+```
+
+And it's started now:
+
+```bash
+$ pebble services
+Service      Startup  Current  Since
+http-server  enabled  active   today at 11:34 UTC
+```
+
+## 6 Add a new layer
+
+Now let's add another layer containing a different HTTP server. Create a new layer:
+
+```bash
+$ echo """\
+summary: Simple layer 2
+
+description: |
+    Yet another simple layer.
+
+services:
+    http-server-2:
+        override: replace
+        summary: demo http server 2
+        command: python3 -m http.server 8081
+        startup: enabled
+""" > $PEBBLE/layers/002-another-server.yaml
+```
+
+This creates another layer containing only one service running an HTTP server listening on a different port 8081. Then let's add this layer to a plan:
+
+```bash
+$ pebble add layer1 $PEBBLE/layers/002-another-server.yaml
+Layer "layer1" added successfully from "/home/ubuntu/PEBBLE_HOME/layers/002-another-server.yaml"
+```
+
+When we update the service configuration by adding a layer, the services changed won't be automatically restarted. If we check the services:
+
+```bash
+$ pebble services
+Service        Startup  Current   Since
+http-server    enabled  active    today at 11:41 UTC
+http-server-2  enabled  inactive  -
+```
+
+We can see that although a new service has been added, it's not active yet.
+
+To bring the service state in sync with the new configuration, run `pebble replan`:
+
+```
+$ pebble replan
+2024-06-02T11:40:39Z INFO Service "http-server" already started.
+```
+
+Now if we check all the services:
+
+```bash
+$ pebble services
+Service        Startup  Current  Since
+http-server    enabled  active   today at 11:34 UTC
+http-server-2  enabled  active   today at 11:40 UTC
+```
+
+We can see that the new HTTP server defined in the newly added layer is brought up as well.
+
+## 7 Where to go from here
+
+- To know more about running the Pebble daemon, see [How to run the daemon (server)](../how-to/run-the-daemon.md).
+- To know more about viewing, starting and stopping services, see [How to view, start, and stop services](../how-to/view-start-stop-services.md).
+- To know more about updating and restarting services, see [How to update and restart services](../how-to/update-restart-services.md).
+- To know more about configuring layers, see [How to configure layers](../how-to/configure-layers.md).
+- To know more about the specification of layers, read the [Layer specification](../reference/layer-specification.md).
+- Read other [how-to articles](../how-to/index.md), [explanations](../explanation/index.md) and [references](../reference/index.md).
```

Comment:
This seems too general too me (and they can just use the menu to get elsewhere). I'd remove this last bullet point.